web/dashboards/garage/js/garage.js

```diff
@@ -1,327 +1,331 @@
+        }
+      }
+    }
+    if(isLab) {
+      if(!isDisappearance){
+        if(!presenceLab.includes(transmitterId)){
+          presenceLab.push(transmitterId);
+          if(isIntern) {
+            internLab.className = "bg-success";
+            cormorant.retrieveStory(storyUrl, function(story){
+            cuttlefish.render(story, internLab);
+            });
+          } 
+          else if(isManager) {
+            managerLab.className = "bg-warning";
+            cormorant.retrieveStory(storyUrl, function(story){
+            cuttlefish.render(story, managerLab);
+            });
+          }
+          else if(isVisitor) {
+            visitorLab.className = "bg-primary";
+            cormorant.retrieveStory(storyUrl, function(story){
+            cuttlefish.render(story, visitorLab);
+            });
+          }
+        }
+      }
+      else{
+        presenceLab.splice(presenceLab.indexOf(transmitterId), 1);
+        if(isIntern) {
+          internLab.removeChild(internLab.childNodes[0]); 
+        } 
+        else if(isManager) {
+          managerLab.removeChild(managerLab.childNodes[0]);  
+        }
+        else if(isVisitor) {
+          visitorLab.removeChild(visitorLab.childNodes[0]);   
+        }
+      }
+    }
+  }
+ }
+
+function dispTime() {
+  let now = new Date();
+}
+
+// Handle environmental beacon data
+function handleEnvironmentalBeacon(raddec) {
+  let packet = raddec.packets[0];
+  //let isSensorPacket = packet && packet.length === 72;
+  let isSensorPacket = packet && packet.length === 64;
+  let valueTemperature;
+
+  if (isSensorPacket) {
+    let t =
+      parseInt(packet.substr(44, 2), 16) +
+      parseInt(packet.substr(46, 2), 16) / 256;
+    let h =
+      parseInt(packet.substr(48, 2), 16) +
+      parseInt(packet.substr(50, 2), 16) / 256;
+    temperature.textContent = t.toFixed(1);
+  }
+}
+// GET the JSON response from the given URL
+function getJson(url, callback) {
+  let httpRequest = new XMLHttpRequest();
+
+  httpRequest.onreadystatechange = function() {
+    if (httpRequest.readyState === XMLHttpRequest.DONE) {
+      if (httpRequest.status === 200) {
+        return callback(JSON.parse(httpRequest.responseText));
+      } else {
+        console.log("Error", httpRequest.status, "on GET", url);
+        return callback(null);
+      }
+    }
+  };
+  httpRequest.open("GET", url);
+  httpRequest.setRequestHeader("Accept", "application/json");
+  httpRequest.send();
+}
+
+// Updare Time
+function updateClock() {
+  let now = new Date();
+  dateNode.innerText = now.toLocaleTimeString("en-US",{ hour12:false });
+}
+
+// Parse the config and start periodic data update 
+function handleConfigAndStart(response) {
+  if(response) {
+    config = response;
+    initialiseBeaver(config.hlcServerUrl);
+  } 
+  else {
+    console.log("Unable to get config.  Refresh page to try again.");
+  }
+}
+
+//refresh occupancy divs
+ let count = 0;
+ function refresh(){

```

Comment:
refreshing of the lab, office and reception divs every 1000s allows showing the right number of Bluetooth beacons detected